lib/ReactViews/Custom/Chart/MomentPointsChart.jsx

```diff
@@ -31,36 +33,37 @@ class MomentPointsChart extends React.Component {
         domain: basisItemScales.y.domain(),
         range: scales.y.domain()
       });
-      const interpolatedPoints = chartItem.points.map(p =>
-        interpolate(p, basisItem.points, basisToSourceScale)
-      );
+      const interpolatedPoints = chartItem.points.map(p => ({
+        ...p,
+        ...interpolate(p, basisItem.points, basisToSourceScale)
+      }));
       return interpolatedPoints;
     }
     return chartItem.points;
   }
 
   doZoom(scales) {
```

Comment:
not part of this PR but we may want to consider better typing of this function either through enforcing it to exist through an interface or through typescript. Right now we kinda expect it to just be there in <LineAndPointChart />

Reply:
Valid point. I would like to simplify some of the chart components at some point and also convert them to `tsx`.